src/api/spec/features/beta/webui/requests_index_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'browser_helper'
+
+RSpec.describe 'Requests Index' do
+  let(:submitter) { create(:confirmed_user, :with_home, login: 'kugelblitz') }
+  let(:receiver) { create(:confirmed_user, :with_home, login: 'titan') }
+  let(:target_project) { receiver.home_project }
+  let(:target_package) { create(:package, name: 'goal', project_id: target_project.id) }
+  let(:source_project) { submitter.home_project }
+  let(:source_package) { create(:package, name: 'ball', project_id: source_project.id) }
+
+  let!(:bs_requests) do
+    create_list(:bs_request_with_submit_action, 6,
+                description: 'Please take this',
+                creator: submitter,
+                target_package: target_package,
+                source_package: source_package)
+
+    create_list(:bs_request, 4,
+                type: 'maintenance_release',
+                source_project: submitter.home_project,
+                creator: submitter)
+
+    create(:set_bugowner_request, creator: submitter)
+
+    create_list(:bs_request_with_maintenance_release_actions, 3,
+                source_project: source_project,
+                state: :review,
+                creator: submitter)
+  end
+
+  context 'request index page' do
+    describe 'show all requests' do
+      before do
+        login submitter
+        Flipper.enable(:request_index)
+        visit requests_path(params: { involvement: 'all' })
+      end
+
+      it 'show all requests' do
```

Comment:
What about request the user is not involved in?

Reply:
I doubt I was involved in that many requests

![Screenshot from 2024-07-23 11-25-16](https://github.com/user-attachments/assets/38f8c9b8-d471-4c40-957d-36fbe4ed880e)

Reply:
```
[1] pry(main)> BsRequest.count
=> 1169686
```

Reply:
IIRC it was noted during the mob session, but probably we didn't fixed in the end.
https://github.com/openSUSE/open-build-service/blob/master/src/api/app/controllers/webui/request_controller.rb#L335 --> this should represent the `/my/tasks#tab-pane-all-requests` content instead, isn't it?

Reply:
Fixed